README.md

```diff
@@ -100,6 +104,7 @@ Send me a mail or create an issue, I will answer ASAP. :+1:
 
 ## License
 
+```
```

Comment:
Why tho, it's not code.

Reply:
I also don't like the code block for the license. We might as well remove it from the readme and add it as a separate file.